File: ruby/fizzbuzz.rb
def fizzbuzz(number)
  return 0 if number.zero?
  return 'FizzBuzz' if (number % 3).zero? && (number % 5).zero?
  return 'Fizz' if (number % 3).zero?
  return 'Buzz' if (number % 5).zero?

  number
end

File: node/fizzbuzz.js
function fizzbuzz (n) {
  if (n === 0) return 0
  if (n % 3 === 0 && n % 5 === 0) return 'FizzBuzz'
  if (n % 3 === 0) return 'Fizz'
  if (n % 5 === 0) return 'Buzz'
  return n
}

module.exports = fizzbuzz
